numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2746,22 +2754,21 @@ def new(
         if config.CUDA_ENABLE_PYNVJITLINK and driver_ver < (12, 0):
             raise ValueError("Enabling pynvjitlink requires CUDA 12.")
         if config.CUDA_ENABLE_PYNVJITLINK:
-            linker = PyNvJitLinker
```

Comment:
`CudaPythonLinker` and `PyNvJitLinker` look like dead code with this change. Is there any reason not to remove them?